log.graphqls

```diff
@@ -82,5 +84,6 @@ enum ContentType {
 }
 
 extend type Query {
+    supportQueryLogsByKeywords: Boolean!
```

Comment:
@arugal @kezhenxu94 @EvanLjp @dmsolr What about this? And I put keywords as List.